### ISSUE TITLE
Cannot delete wallet

### ISSUE THREAD

**Issue by almostheaven1863** [cryptoadvance/specter-desktop#1745]
I recently created a new multisig wallet and transferred everything from old multisig. Despite clicking to delete the old wallet, it continues to show up in the list stating that it is loading forever. Here is a pic of what it looks like and also a pic from when I run the logs.
<img width="1141" alt="Screen Shot 2022-06-04 at 8 26 05 AM" src="https://user-images.githubusercontent.com/98767163/172001315-554235e9-00b0-4675-8371-20658598b8be.png">
<img width="1236" alt="Screen Shot 2022-06-04 at 8 05 57 AM" src="https://user-images.githubusercontent.com/98767163/172001323-ca99a4e9-649c-4ffc-840e-0c939b6e213e.PNG">

**Comment by relativisticelectron** [comment]
The code has changed from version 1.8.1 
https://github.com/cryptoadvance/specter-desktop/blob/eaeac1f20482dbb25ef6867a1a5414df9fae766a/src/cryptoadvance/specter/util/version.py#L81
to 
https://github.com/cryptoadvance/specter-desktop/blob/fa20169b57936404a20bcbe95cc919a8e49d4b3f/src/cryptoadvance/specter/util/version.py#L86

Could you upgrade to the newest  version and check if the error is gone?

**Comment by almostheaven1863** [comment]
I will give this a try. I am using Umbrel so sometimes their update is behind Specter’s. Thanks.

**Comment by relativisticelectron** [comment]
> I will give this a try. I am using Umbrel so sometimes their update is behind Specter’s. Thanks.

Yes, the old version on umbrel is quite annoying.   I try to investigate how to update their docker image: https://github.com/getumbrel/umbrel/issues/1409
